web-extensions/chrome/engine/ui.js

```diff
@@ -99,12 +112,12 @@ BmcUI.prototype.removeSidePanel = function() {
     this._messaging.removeWindowHandlers(this.SIDEPANEL_ID);
 };
 
-BmcUI.prototype.makeTrackingNotification = function(err) {
+BmcUI.prototype.makeNotification = function(operation, err) {
     var evData = {
         type: "action",
         action: "notification",
-        operation: "track",
-        error: err,
+        operation,
+        error: (err||{}).message,
```

Comment:
Won't `{}.message` fail pretty badly?

Reply:
`{}.something` ends up as `undefined`, which is much better than `err.something` if `err` is `undefined` => Error because of the access to a member attribute of undefined.

So, we're effectively defining error as `err.message` if err is defined; `undefined` otherwise :)
small js trick.